kebechet/managers/thoth_advise/thoth_advise.py

```diff
@@ -55,6 +56,18 @@
 # Github and Gitlab events on which the manager acts upon.
 _EVENTS_SUPPORTED = ["push", "issues", "issue", "merge_request"]
 
+ADVISE_ISSUE_TITLE = "Kebechet Advise"
+
+STARTED_ADVISE_COMMENT = "Started advise with id {analysis_id} for {env_name} to get current status, click [here](\
+    https://{host}/api/v1/advise/python/{analysis_id})"
```

Comment:
We have a separate status endpoint at `https://{host}/api/v1/advise/python/{analysis_id}/status`. Both should work fine in this case though, the endpoint used in sources will return results once available. The `/status` endpoint will notify that the analysis was successful (without results). I think the result endpoint should work in this case fine 👍🏻